packages/ui/atomic/create-atomic-result-component/tests/test.spec.ts

```diff
@@ -108,5 +111,45 @@ describe(PACKAGE_NAME, () => {
         }).status
       ).toBe(0);
     });
+
+    describe('health-check', () => {
```

Comment:
would it make sense to extract the common code to a test utils or use [table driven tests](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) with parameterized pkg names?

Reply:
> What part of the common code do you have in mind @mrrajamanickam-coveo?

most of the test code between the 2 pkgs seems the same - hence wondering about reducing duplication

Reply:
Everything that does not contain a 'jest' keyword is refactorable IIRC

@y-lakhdar that'd be a big ask, but maybe it'd be interesting to compare side-by-side having all the e2e for the health-check package in the health-check package?